### PR TITLE
{AKS} add live test for azure cni overlay

### PR DESCRIPTION
**Pull request by tyler-lloyd**
Signed-off-by: Tyler Lloyd <tyler.lloyd@microsoft.com>

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py [hunk 1]
```diff
@@ -3269,6 +3269,36 @@ def test_aks_create_with_windows_gmsa(self, resource_group, resource_group_locat
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='centraluseuap')
```
[the remark this answers is not in the document]
Comment (FumingZhang) [reply src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:3273]: Queued another [pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=57367286&view=results) to test in centraluseuap. Test passed.

[the remark this answers is not in the document]
Comment (tyler-lloyd) [reply src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py]: done 👍

**Review by FumingZhang — APPROVED** [review]
LGTM

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**